core/species.cpp

```diff
@@ -3854,126 +4026,405 @@ void Species::SimplifyTreeSequence(void)
 	
 	// remove redundant sites we added
 	{
-		int ret = tsk_table_collection_deduplicate_sites(&tables_, 0);
+		int ret = tsk_table_collection_deduplicate_sites(&tsinfo.tables_, 0);
 		if (ret < 0) handle_error("tsk_table_collection_deduplicate_sites", ret);
 	}
 	
 	// simplify
 	{
 		EIDOS_BENCHMARK_START(EidosBenchmarkType::k_SIMPLIFY_CORE);
 		
-		tsk_flags_t flags = TSK_SIMPLIFY_FILTER_SITES | TSK_SIMPLIFY_FILTER_INDIVIDUALS | TSK_SIMPLIFY_KEEP_INPUT_ROOTS;
+		// BCH 12/9/2024: Removing TSK_SIMPLIFY_FILTER_INDIVIDUALS here, because we now need to filter the individuals
+		// table ourselves after simplifying all the tree sequences (perhaps in parallel); see SimplifyAllTreeSequences().
+		tsk_flags_t flags = TSK_SIMPLIFY_FILTER_SITES | TSK_SIMPLIFY_KEEP_INPUT_ROOTS;
+		
+		// BCH 12/10/2024: This should still work, with our own node table filtering code.  As Jerome explains, "simplify
+		// will still keep the *edges* that are unary, and that's all that matters. The downstream filtering code you
+		// have just looks to see what nodes have references, and filters out those that are not used in any edges."
 		if (!retain_coalescent_only_) flags |= TSK_SIMPLIFY_KEEP_UNARY;
```

Comment:
I think this is supposed to be `TSK_SIMPLIFY_KEEP_UNARY_IN_INDIVIDUALS`; see #487.

Reply:
I've put a comment in the code quoting your comment here.  This is not related to multichrom at all, so I'm going to postpone dealing with it; one can of worms at a time.  Resolving this thread.